docs/cicd_tools/image_builder.md

```diff
@@ -13,45 +13,35 @@ All functions exposed by this module will use the namespaced prefix:
-| CICD_TOOLS_IMAGE_BUILDER_IMAGE_TAG          | The main image tag to be used. If not provided the 7 first chars of the current repository's git commit hash will be used instead | `""`                 | string | No        |
-| CICD_TOOLS_IMAGE_BUILDER_ADDITIONAL_TAGS    | The additional tags (if any) to be created in array format: ("tag1" "tag2" "latest")                                              | `()`                 | Array  | No        |
-| CICD_TOOLS_IMAGE_BUILDER_LABELS             | The labels (if any) to add to the image being built in array format: ("label1=Value1" "label2=value2")                            | `()`                 | Array  | No        |
-| CICD_TOOLS_IMAGE_BUILDER_BUILD_ARGS         | The build arguments to be provided when building the image (if any) in array format: ("buildarg1=Value1" "buildarg2=value2")      | `()`                 | Array  | No        |
-| CICD_TOOLS_IMAGE_BUILDER_BUILD_CONTEXT      | The build context path to use when building the image.                                                                            | `.`                  | None   | string    | No|
-| CICD_TOOLS_IMAGE_BUILDER_CONTAINERFILE_PATH | The Containerfile path to use when building the image.                                                                            | `Dockerfile`         | None   | string    | No|
-| CICD_TOOLS_IMAGE_BUILDER_QUAY_EXPIRE_TIME   | The expire time value to be set for Quay expires labels, used only in change request contexts.                                    | `3d`                 | string | No        |
-| CICD_TOOLS_IMAGE_BUILDER_QUAY_USER          | The username to use when logging in to Quay.io                                                                                    | `$QUAY_USER`         | string | No        |
-| CICD_TOOLS_IMAGE_BUILDER_QUAY_PASSWORD      | The password to use when logging in to Quay.io                                                                                    | `$QUAY_TOKEN`        | string | No        |
-| CICD_TOOLS_IMAGE_BUILDER_REDHAT_USER        | The username to use when logging in to the Red Hat Registry                                                                       | `$RH_REGISTRY_USER`  | string | No        |
-| CICD_TOOLS_IMAGE_BUILDER_REDHAT_PASSWORD    | The password to use when logging in to the Red Hat Registry                                                                       | `$RH_REGISTRY_TOKEN` | string | No        |
-
-The only required variable is `CICD_TOOLS_IMAGE_BUILDER_IMAGE_NAME`, the rest of them are optional
+| Variable name                         | Description                                                                                                                       | Default value        | Type   | Mandatory |
```

Comment:
I'm looking a diff so maybe you have this elsewhere and I just don't see it in diff view (github PR reviews aren't great for working on docs collaboritvely.) BUT, some examples would be helpful. I always like to see examples over lists.